### PR TITLE
test: add code snippets for using bigframes.ml

### PR DESCRIPTION
**Pull request by ashleyxuu**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by ashleyxuu** [comment]
The code snippets here are prepared for the content of "Use BigQuery DataFrames" page.
Content: https://docs.google.com/document/d/1N5mbfoTh63SwTRFkqs-j9m_2WIykDKHPKAECvKScDSg/edit#heading=h.nq1i22bqeo4y

File: samples/snippets/clustering_model_test.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+def test_clustering_model():
+    from bigframes.ml.cluster import KMeans
+    import bigframes.pandas as bpd
+
+    # Load data from BigQuery
+    query_or_table = "bigquery-public-data.ml_datasets.penguins"
+    bq_df = bpd.read_gbq(query_or_table)
+    # [START bigquery_dataframes_clustering_model]
+    # Create the KMeans model
+    cluster_model = KMeans(n_clusters=10)
```
Comment (tswast) [inline samples/snippets/clustering_model_test.py:25]: This sample isn't really "standalone". Needs imports at a minimum.

File: samples/snippets/regression_model_test.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def test_regression_model():
+    import bigframes.pandas as bpd
+
+    query_or_table = "bigquery-public-data.ml_datasets.penguins"
```
Comment (tswast) [inline samples/snippets/regression_model_test.py:19]: This is a lot of setup. Might be best to combine with samples/snippets/clean_and_prep_ml_data_test.py

These aren't really "standalone" samples, as they require so much setup that we aren't showing the user.

**Review by tswast — COMMENTED** [review]
My main concern with merging this PR is that running some of these samples may take a long time. It might be best to create a separate folder for these and exclude them from our presubmit samples tests and move the ML samples tests to the e2e job.

File: samples/snippets/regression_model_test.py (hunk 2 above)
Comment (ashleyxuu) [reply samples/snippets/regression_model_test.py]: Done making it a single region and deleting "clean_and_prep_ml_data_test"

**Review by tswast — APPROVED** [review]
Thanks! These are great.

**Pull request merged**